web_src/css/markup/content.css

```diff
@@ -585,3 +586,25 @@
 .file-view.markup.orgmode li.indeterminate > p {
   display: inline-block;
 }
+
+.markup .file-preview-box {
+  margin-bottom: 16px;
+}
+
+.markup .file-preview-box div:nth-child(1) {
+  padding: .5rem;
+  padding-left: 1rem;
+  border: 1px solid var(--color-secondary);
+  border-bottom: none;
+  border-radius: 0.28571429rem 0.28571429rem 0 0;
+  background: var(--color-box-header);
+}
```

Comment:
`:nth-child(1)` selector is too unstable in regards to future changes, give it a `class` instead.